dir.props

```diff
@@ -0,0 +1,307 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Condition="Exists('..\dir.props')" Project="..\dir.props" />
```

Comment:
It is always a trade-off of how many things to split up. If there are logical groups then yes it make sense to split them but we should also not just have a props file for every property.